glusterd2/commands/peers/deletepeer.go

```diff
@@ -22,8 +22,8 @@ func deletePeerHandler(w http.ResponseWriter, r *http.Request) {
 	logger := gdctx.GetReqLogger(ctx)
 
 	id := mux.Vars(r)["peerid"]
-	if id == "" {
-		restutils.SendHTTPError(ctx, w, http.StatusBadRequest, "peerid not present in the request", api.ErrCodeDefault)
+	if uuid.Parse(id) == nil {
+		restutils.SendHTTPError(ctx, w, http.StatusBadRequest, "Invalid peerid passed in deletePeer request", api.ErrCodeDefault)
```

Comment:
This is error message returned to user. `deletePeer` seems like a variable name or a function name. Use `delete peer` or even something generic like - `invalid peer id passed`. Use already knows which request was it. Same comment below.